test/functionalTest/cases/2248_forwarding_updates_overwrite_the_types_of_attributes/forwarding_updates_overwrite_the_types_of_attributes.test

```diff
@@ -139,12 +141,18 @@ Date: REGEX(.*)
 
 
 03. Update E1/T1/A1 in CB, using PUT /v2/entities/E1/attrs/A1/value, without stating type of the attribute, value is a string
+Note: we get 404 Not Found, but is ok. It is due to accumulator-server.py doesn't answers in NGSIv1
 =============================================================================================================================
-HTTP/1.1 204 No Content
-Content-Length: 0
+HTTP/1.1 404 Not Found
+Content-Length: 61
+Content-Type: application/json
 Fiware-Correlator: REGEX([0-9a-f\-]{36})
 Date: REGEX(.*)
 
+{
```

Comment:
This change in .test (and some other similar) corresponds to the following entry in CNR: "Fix: using 404 Not Found - NotFound error when  NGSIv2 forwarded update (completely) fails (previously 204 No Content was used, without providing actual informaton about the partial update)".

This is a bit surprising... not sure if the statement above is correct in all cases on only in the case the CPr response but in a not-NGSIv1 way (as the accumulator-server.py does). 

@kzangeli , do we have some .test covering NGSIv2 update fwd in the case of 1) CPr not responding , 2) CPr responding. We should check **with the release/1.2.0 code** what the client get in that case (either 404 Not Found or 204 No Content).